workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandRequestSender.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Improbable.Gdk.Core.Commands;
+using Improbable.Worker;
+using Unity.Entities;
+using UnityEngine.Assertions.Must;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
+    {
+        [InjectableId(InjectableType.WorldCommandRequestSender, InjectableId.NullComponentId)]
+        [InjectionCondition(InjectionCondition.RequireNothing)]
+        public class WorldCommandRequestSender : IInjectable
+        {
+            private readonly Entity entity;
+            private readonly EntityManager entityManager;
+
+            private WorldCommandRequestSender(Entity entity, EntityManager entityManager)
+            {
+                this.entity = entity;
+                this.entityManager = entityManager;
+            }
+
+            public WorldCommands.ReserveEntityIds.Request ReserveEntityIds(
+                uint numberOfEntityIds,
+                object context = null)
+            {
+                var reserveEntityIdCommandSender =
+                    entityManager.GetComponentData<WorldCommands.ReserveEntityIds.CommandSender>(entity);
+
+                var request = new WorldCommands.ReserveEntityIds.Request
```

Comment:
Prefer to use the CreateRequest functions I think.

Reply:
Yes this is a bug, fixing

Reply:
Also missing timeoutmillis